spec/requests/api/v1/mentor_post_request_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe 'mentors API' do
+  describe 'POST /api/v1/mentors/' do
+    it 'posts successfully to the db' do
+
+      payload = {
+        name: "Atreyu",
+        email: "never_ending@gmail.com",
+        city: "New York",
+        state: "NY",
+        slack_username: "atreyu@slack.com",
+        linkedin_username: "atreyu@linked_in.com",
+        github_username: "atreyu@github.com",
+        program: "FE",
+        matched: false,
+        active: true
+      }
+
+      post '/api/v1/mentors', params: {mentor: payload}
+
+      mentor = Mentor.last
+
+      expect(response).to be_successful
+      expect(response.status).to eq(200)
+      expect(mentor.name).to eq(payload[:name])
+
+    end
+    it 'does not post successfully with missing attributes' do
```

Comment:
I am fine with not building controller functionality / a test for POST having missing mentor attributes right now because 1. we WILL write a test for a sad path for a missing / incorrect JSON auth token when we finish implementing that branch, and 2. @caseyhoepner @raualex The front end will have the responsibility of not allowing the mentor's account form to be submitted until all required fields are complete.